assets/blocks/quiz/quiz-block/quiz-edit.js

```diff
@@ -60,9 +58,7 @@ const QuizEdit = ( props ) => {
 					'sensei-lms/quiz-question',
 					'sensei-lms/quiz-category-question',
 				] }
-				template={
-					isPostTemplate ? [ [ 'sensei-lms/quiz-question', {} ] ] : []
-				}
+				template={ [] }
```

Comment:
I think we could remove this prop entirely, right?

Reply:
Removed in 2b7c932